be/src/http/http_common.h

```diff
@@ -75,5 +75,6 @@ static const std::string HTTP_PARTIAL_UPDATE_MODE = "partial_update_mode";
 
 static const std::string HTTP_100_CONTINUE = "100-continue";
 static const std::string HTTP_CHANNEL_ID = "channel_id";
+static const std::string HTTP_COMPRESSION = "compression";
```

Comment:
not suggest go this way. The standard Content-Encoding is doing exactly the same thing, customized header should be used as few as possible.

Reply:
I agree. @jaogoy PTAL.

Reply:
compression or compression_type